kola/tests/kubeadm/templates.go

```diff
@@ -307,8 +317,23 @@ EOF
     cp /etc/kubernetes/admin.conf /home/core/.kube/config
     chown -R core:core /home/core/.kube; chmod a+r /home/core/.kube/config;
 
+{{ if eq .CNI "calico" }}
     kubectl create -f https://docs.projectcalico.org/manifests/tigera-operator.yaml
     kubectl apply -f calico.yaml
+{{ end }}
+{{ if eq .CNI "flannel" }}
+    curl -sSL https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml > kube-flannel.yml
```

Comment:
```suggestion
    curl -sSfL https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml > kube-flannel.yml
```
We should fail on 404